pkg/apis/config/v1alpha4/default.go

```diff
@@ -62,6 +65,9 @@ func SetDefaultsCluster(obj *Cluster) {
 		if obj.Networking.IPFamily == "ipv6" {
 			obj.Networking.ServiceSubnet = "fd00:10:96::/112"
 		}
+		if obj.Networking.IPFamily == "DualStack" {
+			obj.Networking.ServiceSubnet = "10.96.0.0/12,fd00:10:96::/112"
```

Comment:
@aojea I am currently working on Calico dual stack support.  Your `dualstack` branch works great. The only issue I came across is ServiceSubnet `fd00:10:96::/112` is overlapping with PodSubnet `fd00:10:244::/16`. This causes host accessing service ip failed to hit MASQ rule and the remote host saw service ip as source address. Using ServiceSubnet `fa00:10:96::/112` get this issue fixed.

Reply:
yeah, good catch, this was an oversight on my part.
This was because of a limitation that was fixed now with this https://github.com/kubernetes/kubernetes/pull/79993

Reply:
it should be working now, I've updated the PR

Reply:
Hi @aojea , with your latest code, and kubernetes 1.17.0 node image, I hit an issue creating cluster

Reply:
the API version has changed, I'm deploying it know with this
```
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
networking:
  ipFamily: DualStack
nodes:
- role: control-plane
- role: worker
- role: worker
kubeadmConfigPatches:
- |
  apiVersion: kubeadm.k8s.io/v1beta2
  kind: ClusterConfiguration
  metadata:
    name: config
  featureGates:
    IPv6DualStack: true
```

Reply:
Thanks! Sure, I will let you know the result today.

Reply:
@aojea With API version update, I can create dual stack cluster correctly. Dual stack pod IP/service IP all works fine for me. ( I am using lastest Calico ). However, I can only see ipv4 address been picked up for node resources. The kind node got both ipv4 and ipv6 address assigned to eth0. Do you know what possible goes wrong?

Reply:
@song-jiang thanks for confirming, appreciate your help.
The kubelet only reports one ip family without a cloud-controller, that's a known issue 
https://github.com/kubernetes/kubernetes/issues/42125
it's not clear what direction is going to take kubernetes regarding this, currently seems that the only solution is with an external cloud-controller or patching manually 🤷‍♂

Reply:
@aeojea Good to know. Thanks!